service/popm/popm.go

```diff
@@ -332,6 +338,30 @@ func (s *Server) latestKeystones(ctx context.Context, count int) (*gethapi.L2Key
 	return &kr, nil
 }
 
+func (s *Server) getFee(ctx context.Context) (*tbcapi.FeeEstimate, error) {
+	log.Tracef("getFee")
+	defer log.Tracef("getFee exit")
+
+	var feeAmount *tbcapi.FeeEstimate
+	if s.cfg.StaticFee != 0 {
+		feeAmount = &tbcapi.FeeEstimate{
+			Blocks:      s.cfg.BitcoinConfirmations,
+			SatsPerByte: float64(s.cfg.StaticFee),
+		}
+	} else {
+		// Estimate BTC fees.
+		feeEstimates, err := s.gozer.FeeEstimates(ctx)
+		if err != nil {
+			return nil, fmt.Errorf("fee estimates: %w", err)
+		}
+		feeAmount, err = gozer.FeeByConfirmations(s.cfg.BitcoinConfirmations, feeEstimates)
+		if err != nil {
+			return nil, fmt.Errorf("fee by confirmations: %w", err)
+		}
+	}
+	return feeAmount, nil
```

Comment:
I think this can be simplified as:
```suggestion
	if s.cfg.StaticFee != 0 {
		return  &tbcapi.FeeEstimate{
			Blocks:      s.cfg.BitcoinConfirmations,
			SatsPerByte: float64(s.cfg.StaticFee),
		}
	}

	// Estimate BTC fees.
	feeEstimates, err := s.gozer.FeeEstimates(ctx)
	if err != nil {
		return nil, fmt.Errorf("fee estimates: %w", err)
	}
	feeAmount, err := gozer.FeeByConfirmations(s.cfg.BitcoinConfirmations, feeEstimates)
	if err != nil {
		return nil, fmt.Errorf("fee by confirmations: %w", err)
	}
	return feeAmount, nil
```

Reply:
Oops, you're right! What I get for copy pasting stuff around